src/Sylius/Bundle/AddressingBundle/EventListener/ZoneMemberIntegrityListener.php

```diff
@@ -71,7 +71,10 @@ private function getSession(): SessionInterface
     {
         // bc-layer for Symfony 4
         if (!method_exists(RequestStack::class, 'getSession')) {
-            return $this->requestStack->getMasterRequest()->getSession();
+            $request = $this->requestStack->getMasterRequest();
+            Assert::notNull($request, 'No main request available to get a session !');
```

Comment:
```suggestion
            Assert::notNull($request, 'No main request available to get a session!');
```

Reply:
Ah yes my french syntax is always following me :D